inst/@sym/jn.m

```diff
@@ -0,0 +1,61 @@
+%% Copyright (C) 2016 Utkarsh Gautam 
+%%
+%% This file is part of OctSymPy.
+%%
+%% OctSymPy is free software; you can redistribute it and/or modify
+%% it under the terms of the GNU General Public License as published
+%% by the Free Software Foundation; either version 3 of the License,
+%% or (at your option) any later version.
+%%
+%% This software is distributed in the hope that it will be useful,
+%% but WITHOUT ANY WARRANTY; without even the implied warranty
+%% of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See
+%% the GNU General Public License for more details.
+%%
+%% You should have received a copy of the GNU General Public
+%% License along with this software; see the file COPYING.
+%% If not, see <http://www.gnu.org/licenses/>.
+
+%% -*- texinfo -*-
+%% @documentencoding UTF-8
+%% @deftypefn  {Function File} {@var{Y} =} jn (@var{alpha}, @var{x})
+%% Symbolic Spherical Bessel function of the first kind.
+%%
+%% Example:
+%% @example
+%% @group
+%% syms n x
+%% A = 2*jn(n, x)
```

Comment:
drop this 2 (I sometimes use that to avoid a divide by 2 in the result, but here it doesn't seem its needed.